frontend/src/app/superadmin/users/users.component.ts

```diff
@@ -158,10 +151,7 @@ export class UsersComponent implements OnInit {
   }
 
   deleteObject(): void {
-    let selectedRows = this.tableselectionCheckbox.selected;
```

Comment:
Funktioniert dein Linter? Hier meckert er (zurecht) bei mir:
"ESLint: 'selectedRows' is never reassigned. Use 'const' instead.(prefer-const)"